src/id3/v2/tag.rs

```diff
@@ -547,21 +607,50 @@ impl SplitAndMergeTag for ID3v2Tag {
 		fn split_pair(
 			content: &str,
 			tag: &mut Tag,
-			current_key: ItemKey,
+			number_key: ItemKey,
 			total_key: ItemKey,
 		) -> Option<()> {
+			fn parse_number(source: Option<&str>) -> Option<&str> {
```

Comment:
This function should accept `source: &str`. Otherwise you are implicitly re-implementing `Option::and_then`.